src/Models/Device.cs

```diff
@@ -21,6 +21,9 @@ public class Device
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "push_provider")]
         public string PushProvider { get; set; }
 
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "push_provider_name")]
+        public string PushProviderName { get; set; }
+
```

Comment:
it seems names in the line 7 are missing huawei and xiaomi

Reply:
this applies the field in line 21 (but wrote here since there is no diff there)

Reply:
we can remove them, use app level ones

Reply:
I went ahead and removed them. they weren't used